backend/app.server.js

```diff
@@ -103,20 +99,11 @@ fetchRooms(ROOMS_SOURCE)
       console.log(roomsData);
 
       app.locals.roomsDetail = roomsData;
-      
+
     })
     .catch(err => {
       console.error(err);
 });
 
 
-// Listen on port 8080
-const server = app.listen(PORT, HOST);
-console.log(`Running on http://${HOST}:${PORT}`);
-
-const officeControllerInstance = new OfficeController();
-const office = new Office(officeControllerInstance, server);
-
-office.start();
-
-module.exports = server;
+module.exports = app;
```

Comment:
Surely this is a great way to organize this code. Now with your feedback I feel more comfortable doing this refactoring.
I answer again when it's ready.

Reply:
@alesshh I did a little different from what you suggested. I have separated in the following files:

- app.config.js - isolate the configuration constants
- app.routes.js - implementation of app routes
- app.server.js - build the server rules
- index.js
- office.factory.js - as you suggested in the other comment, I put the socket construction here
- office.socket.js - the socket logic

just changing the file name to _routes.js_ did not make much sense because of the over responsibility that was in it. Since the design pattern was _name.pattern.js_, I thought it would make sense to stick to the pattern and I separated it.